libqtile/widget/volume.py

```diff
@@ -185,16 +190,22 @@ def get_volume(self):
             if self.get_volume_command:
                 get_volume_cmd = self.get_volume_command
 
-            mixer_out = self.call_process(get_volume_cmd)
+            mixer_out = subprocess.getoutput(get_volume_cmd)
         except subprocess.CalledProcessError:
             return -1
 
-        if "[off]" in mixer_out:
+        check_mute = mixer_out
+        if self.check_mute_command:
+            check_mute = subprocess.getoutput(self.check_mute_command)
+
+        if self.check_mute_string in check_mute:
             return -1
 
         volgroups = re_vol.search(mixer_out)
         if volgroups:
             return int(volgroups.groups()[0])
+        elif "pamixer" in self.get_volume_command:
+            return mixer_out
```

Comment:
I'm not sure about these lines.

Firstly, the method is mean to return an integer but `subprocess.getoutput` returns a string.
Secondly, even if you convert the output to an integer, I suspect that there are other `pamixer` commands that wouldn't give the right output.
Thirdly, what's special about `pamixer` that means we only do this check for that application?